examples/with-chakra-ui-typescript/package.json

```
@@ -3,15 +3,15 @@
   "scripts": {
     "dev": "next",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "lint": "next lint"
```

```suggestion
    "start": "next start"
```

Since ESLint is not part of this example, it might be confusing to add this script.